src/wallet/rpcwallet.cpp

```diff
@@ -3979,7 +3979,7 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
                                     {"vout_index", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "The zero-based output index, before a change output is added."},
                                 },
                             },
-                            {"replaceable", RPCArg::Type::BOOL, /* default */ "false", "Marks this transaction as BIP125 replaceable.\n"
+                            {"replaceable", RPCArg::Type::BOOL, /* default */ "fallback to wallet's default", "Marks this transaction as BIP125 replaceable.\n"
```

Comment:
"fallback to" seems unnecessary here.

Reply:
Would need a scripted diff to change that?

```
$ git grep --count  "fallback to wallet's default" 
src/wallet/rpcwallet.cpp:7

Reply:
I used the same wording as elsewhere, so could indeed to a replace-all.

Reply:
Done, but not scripted.